express-validator/express-validator.d.ts

```diff
@@ -85,39 +108,44 @@ declare namespace ExpressValidator {
 		/**
 		 * isNumeric accepts zero padded numbers, e.g. '001', isInt doesn't
 		 */
-		isInt(): Validator;
+		isInt(options?: MinMaxOptions): Validator;
 		isLowercase(): Validator;
 		isUppercase(): Validator;
 		isDecimal(): Validator;
 		/**
 		 * Alias for isDecimal
 		 */
 		isFloat(): Validator;
+		isFullWidth(): Validator;
+		isHalfWidth(): Validator;
+		isVariableWidth(): Validator;
 		/**
 		 * Check if length is 0
 		 */
-		notNull(): Validator;
+		//notNull(): Validator; // I don't see this in the documentation or code anywhere.
```

Comment:
I think that fork is just an out of date version of validator. The npm package express-validator depends on validator (https://github.com/chriso/validator.js) so these updates look appropriate to me.

I'll merge this if no objections from @ctavan / @axefrog /@dreampulse